dist/obs-server.spec

```diff
@@ -225,6 +225,8 @@ Requires:       user(obsservicerun)
 Requires:       zstd
 # needed for optional sqlite databases, which are default for new installations
 Requires:     perl-DBD-SQLite
+# psmisc for obsscheduler script
+Requires:       psmisc
```

Comment:
```suggestion
```
Is `psmisc` still needed after changing `checkproc` to `pgrep` and `killall` to `pkill`?

Reply:
I mean, `pgrep` and `pkill` are provided by `procps`, which is already required.